.github/workflows/build-test-measure.yml

```diff
@@ -6,29 +6,65 @@ on:
       - develop
       # Include all release branches.
       - '[0-9]+.[0-9]+'
-    # Don't run for irrelevant changes.
-    paths-ignore:
-      - '!.github/workflows/*'
-      - '.wordpress-org/**'
-      - '!bin/ci/**'
-      - '!bin/local-env/**'
-      - 'docs/**'
-      - 'qa-tester/**'
   pull_request:
     # Run workflow whenever a PR is opened, updated (synchronized), or marked ready for review.
     types: [opened, synchronize, ready_for_review]
-    # Don't run for irrelevant changes.
-    paths-ignore:
-      - '!.github/workflows/*'
-      - '.wordpress-org/**'
-      - '!bin/ci/**'
-      - '!bin/local-env/**'
-      - 'docs/**'
-      - 'qa-tester/**'
 
 jobs:
+  pre-run:
+    name: 'Pre run'
+    runs-on: ubuntu-latest
+    outputs:
+      changed-file-count: ${{ steps.determine-file-count.outputs.count }}
```

Comment:
Idea: what if there was a `changed-php-file-count`, `changed-css-file-count`, and `changed-js-file-count`? Then `lint-css` could, for example, only run if `needs.pre-run.outputs.changed-css-file-count > 0`. Same for the others. This would speed up the runs a bit.

Reply:
Me gusta. I'll make the change.